### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by RobAtticus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # TimescaleDB Changelog
 
+## 0.5.0 (2017-09-20)
+
+**High-level changes**
+* New histogram function added for getting the frequency of a column's values. For detailed usage, see [our docs](http://docs.timescale.com/api#advanced-analytics).
+* Add support for using `DATE` as partition column.
+* `chunk_time_interval` now supports `INTERVAL` data types
+* Block several unsupported and/or dangerous operations on hypertables and chunks, including dropping or otherwise altering a chunk directly.
+* Improved support for constraints on hypertables.
```
Comment (mfreed) [inline CHANGELOG.md]: Improved support for primary keys, foreign keys, and constraints on hypertables.

^ Also, this is probably key feature of release.  Move to bullet #1?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
 # TimescaleDB Changelog
 
+## 0.5.0 (2017-09-20)
+
+**High-level changes**
+* New histogram function added for getting the frequency of a column's values. For detailed usage, see [our docs](http://docs.timescale.com/api#advanced-analytics).
```
[the remark this answers is not in the document]
Comment (RobAtticus) [reply CHANGELOG.md]: Is the next docs release going to coincide with this release?

[the remark this answers is not in the document]
Comment (mfreed) [reply CHANGELOG.md]: Most of required docs changes, including the histogram, are sitting in PRs.  We can get both out tomorrow.

File: CHANGELOG.md (hunk 1 above)
Comment (cevian) [reply CHANGELOG.md]: Maybe "Improved support for primary-key, foreign-key, unique, and exclusion constraints"

**Pull request merged**